docs/importing.md

```diff
@@ -0,0 +1,78 @@
+# Importing existing Data into Chronicle
+
+## Importing Data into Chronicle
+
+Chronicle provides a CLI command `import` to load data from a JSON-LD file that
+contains an array of Chronicle Operations. These operations can be used to
+create agents, namespaces, and other resources in the Chronicle database. For
+more information about the data format of the import file, see [Recording Provenance](../cli#import-namespace-id-namespace-uuid-url).
```

Comment:
```suggestion
more information about the data format of the import file, see [Recording Provenance](recording_provenance.md#importing-data-into-chronicle).
```
Sorry, @alexhq, found another thing: The info about the data format etc is in Recording Provenance in the Importing Data into Chronicle section.

Reply:
Ah thanks, I've corrected that now